packages/client/src/utils/contract.ts

```diff
@@ -139,8 +139,16 @@ export const createDecoratedContract = <T extends Contract>(
     const result: any = {
         eventEmitter
     }
+
+    function getAllPropertyNames(obj: object): string[] {
```

Comment:
This change is to improve browser compatibility? Could mention this change in the PR description (just in case this introduces some regression which we need to debug later)

Reply:
Added mention to PR